File: Model/SaveModel.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Isocline.Model
{
    internal class SaveModel
    {
        private PictureBox picture;
        public SaveModel(PictureBox picture)
        {
            this.picture = picture;
        }

        public void Save(string filePath)
        {
            picture.Image.Save(filePath);
        }
    }
}

File: Model/SaveModel.vb
Imports System.Windows.Forms

Namespace Isocline.Model
    Friend Class SaveModel
        Private picture As PictureBox
        Public Sub New(ByVal picture As PictureBox)
            Me.picture = picture
        End Sub

        Public Sub Save(ByVal filePath As String)
            picture.Image.Save(filePath)
        End Sub
    End Class
End Namespace
